src/nodesFactory/ISimpleNodesFactory.ts

```diff
@@ -0,0 +1,21 @@
+import { Attr, CDATASection, Comment, Element, ProcessingInstruction, Text } from '../types/Types';
+
+/**
+ * Subset of the constructor methods present on Document. Can be used to create textnodes, elements,
+ * attributes etcetera.
```

Comment:
Can we just write down all nodes you can create instead of using `etcetera`? They are not that many so we might as well, e.g. `Subset of the constructor methods present on Document. Can be used to create attributes, CDATA sections, comments, elements, processing instructions, and text nodes.`